reporters/SendSlackReport.sh

```diff
@@ -22,7 +22,7 @@ echo "${reportOutput}" >$fileName
 
 # Spesify the parameters to send to the slack API. All caps values need to be set as enviroment variables.
 # Note that this SLACK_TOKEN is an API token, not a webhook.
-messageTitle=$date $BOT_IDENTIFIER " Daily Report"
+messageTitle="$date $BOT_IDENTIFIER Daily Report"
```

Comment:
Before this, I was getting an error `../reporters/SendSlackReport.sh: line 25: nick: command not found` where my environment was `BOT_IDENTIFIER=nick`